miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSubprojectService.java

```diff
@@ -26,7 +26,7 @@
   protected static final Logger log = LoggerFactory.getLogger(DefaultSubprojectService.class);
 
   @Autowired
-  private SQLProjectDAO sqlProjectDAO;
+  private HibernateProjectDao sqlProjectDAO;
```

Comment:
Actually, it looks like those bean definitions in db-config.xml were removed. That should also be fine now, as long as DAOs are marked with `@Repository` I think